plugins/woo-ai/src/image-background-removal/index.tsx

```diff
@@ -1,19 +1,16 @@
-/**
- * External dependencies
- */
-// eslint-disable-next-line @typescript-eslint/ban-ts-comment
-// @ts-ignore createRoot included for future compatibility
-// eslint-disable-next-line @woocommerce/dependency-group
-import { render, createRoot } from '@wordpress/element';
-
 /**
  * Internal dependencies
  */
 import { BackgroundRemovalLink } from './background-removal-link';
 import { getCurrentAttachmentDetails } from './image_utils';
 import { FILENAME_APPEND, LINK_CONTAINER_ID } from './constants';
+import { renderWrappedComponent } from '../utils';
```

Comment:
Since this file doesn't have any JSX in it, we could rename it to `index.ts`, right?